target/sama5d2/chip_pins.h

```diff
@@ -856,7 +856,7 @@
 
 /* QSPI0 IOSET 3 */
 #define PINS_QSPI0_IOS3 {\
-	{ PIO_GROUP_A, 0x0fc00000, PIO_PERIPH_F, PIO_PULLUP },\
+	{ PIO_GROUP_A, 0x0fc00000, PIO_PERIPH_F, PIO_PULLUP | PIO_DRVSTR_HI },\
```

Comment:
Your proposed modification of the DRVSTR is hardcoded in the chip_pins for this IOSET. We would prefer a supplementary function in pio to set DRVSTR. When configuring the pio in the main functino, you can set here the DRVSTR high and then configure the QSPI with the defined high frequency.